.github/workflows/benchmarking.yml

```diff
@@ -56,16 +56,23 @@ concurrency:
 jobs:
   bench:
     if: ${{ github.event.inputs.run_only_pgvector_tests == 'false' || github.event.inputs.run_only_pgvector_tests == null }}
+    permissions:
+      contents: read
+      id-token: write # Required for OIDC authentication in azure runners
     strategy:
       fail-fast: false
       matrix:
         include:
           - DEFAULT_PG_VERSION: 16
             PLATFORM: "neon-staging"
             region_id: ${{ github.event.inputs.region_id || 'aws-us-east-2' }}
+            RUNNER: [ self-hosted, us-east-2, x64 ]
+            IMAGE: 369495373322.dkr.ecr.eu-central-1.amazonaws.com/build-tools:pinned
           - DEFAULT_PG_VERSION: 16
             PLATFORM: "azure-staging"
             region_id: 'azure-eastus2'
+            RUNNER: [ self-hosted, eastus2, azure ]
```

Comment:
optional: This could be similar to the other one, as in `[ self-hosted, eastus2, x64 ]`, `azure` is in there just in case we need to differentiate between them at some point, without depending on region names. Practically doesn't make a difference though.

Reply:
ok, didn't know that we have the x64 label for azure runners, too